src/core/notifier/handler/webhook/http_handler.go

```diff
@@ -0,0 +1,57 @@
+package webhook
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/goharbor/harbor/src/common/job/models"
+	"github.com/goharbor/harbor/src/jobservice/job"
+	"github.com/goharbor/harbor/src/webhook"
+	"github.com/goharbor/harbor/src/webhook/hook"
+)
+
+// HTTPScheduler ...
```

Comment:
more comment please.
Additionally, the file name is handler but inside it's called scheduler

Reply:
> more comment please.
> Additionally, the file name is handler but inside it's called scheduler

will fix it